src/core/CoolBoard.cpp

```diff
@@ -595,3 +593,39 @@ void CoolBoard::mqttCallback(char *topic, byte *payload, unsigned int length) {
   temp[length] = '\0';
   this->update(temp);
 }
+
+bool CoolBoard::mqttsConfig(){
+if (SPIFFS.exists("/certificate.bin") && SPIFFS.exists("/privateKey.bin")) {
+    File CertificateBin = SPIFFS.open("/certificate.bin", "r");
+    String CertficateString = CertificateBin.readString();
+    CertificateBin.close();
+    uint8_t binaryCert[CertficateString.length()];
+    uint16_t len = b64decode(CertficateString, binaryCert);
```

Comment:
Following the DRY principle, you should write a function that takes a filename as parameter, and outputs a structure containing the `uint8_t` array and its length